study.md

```diff
@@ -43,5 +43,6 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+Hash urls rely on JS parsing to determine what content should be shown. The problem here is that if JS breaks, the site will not show the content. When deploying to GitHub pages, there are a few extra steps, like copying the app build into the root directory. I'm not sure what this last question means.
```

Comment:
Yes Hash urls are problematic is JS breaks, as well as, they don't copy/paste.  If you want to use history API, then you can't deploy on github, you would need deploy your ember app to Heroku. To host an Ember application on github use `ember build --environment production` to build the project to `dist/`, because it is unable to run a server.